src/utilities/parseError.ts

```diff
@@ -0,0 +1,19 @@
+import { inspect } from 'util';
+
+/**
+ * Helper function to help the message part of the type Error
+ */
+
```

Comment:
We normally don't keep lines between the comment and function. ESLint should warn tho 🤔

Reply:
Weird I didn't get warned by it. Have removed the line.